phoenix-core/src/main/java/org/apache/phoenix/cache/ServerCacheClient.java

```diff
@@ -310,7 +311,7 @@ public TaskExecutionMetricsHolder getTaskExecutionMetric() {
                         }
                     }));
                 } else {
-                    if (LOG.isDebugEnabled()) {LOG.debug(addCustomAnnotations("NOT adding cache entry to be sent for " + entry + " since one already exists for that entry", connection));}
```

Comment:
This generates a log line for every single region (not just unique regionservers) that aren't sent a cache. For large tables, this makes `debug` level logs _very_ noisy, even for testing. However, if phoenix tries to avoid making backwards incompatible changes to debug level logs, I can remove this from the PR